.github/workflows/knative-boilerplate.yaml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 The Knative Authors.
```

Comment:
I think you want to set this up in https://github.com/knative-sandbox/knobots/blob/main/repos.yaml so this gets updated, but I guess there's no harm landing an initial version in the mean time.